### ISSUE TITLE
REPL crashes on Windows newlines

### ISSUE THREAD

**Issue by wulmer** [RustPython/RustPython#11]
When I execute `cargo run` and type `1<ENTER>` on the REPL, the parser crashes:

```
Welcome to the magnificent Rust Python 0.0.1 interpreter
>>>>> 1
thread 'main' panicked at 'Not impl Some('\r')', parser\src\lexer.rs:582:21
```

This is somehow related to Windows newlines (`\r\n`), so I'll try to come up with a PR which fixes this issue on Windows.

**Comment by windelbouwman** [comment]
@wulmer Can this issue be closed since you implemented this functionality?

**Comment by wulmer** [comment]
Sure. Was fixed in #15 .
